icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
So we _want_ the alias which is currently called 'author_portrait' to be available to all apps and templates within a project. ie we wanted to use the 'author_portrait' alias from within any template in the project, and we'd expect it to be the same size.

Namespacing an alias makes it only available to the app that namespaces it, which means we couldn't use an author_image anywhere else ie in a listing of authors that a project needs. Easy thumbs doesn't fallback to used a namespaced alias if no global alias is defined. So I can't see a way round starting with a globally namespaced definition, at least without patching easy_thumbnails.

(If another app supplied a template that had a thumbnail that used the same alias, and we wanted it to be a different size, _then_ we would then define a namespaced alias in settings, or override the template.)

Which leaves us with the issue of whether 'author_portrait' is likely to unintentionally clash with another app's alias, and whether encoding w x h in the alias name is a good solution for avoiding clashes.

My feeling is 'author_portrait' is already unlikely to clash with anything else we install. If it does, and we need two different 'author_portrait's for two different purposes, we can define a namespaced alias for the one that isn't ICEkit. However, I'm happy amend it to 'icekit_authors_portrait_large', to encode both the defining app and the purpose, to eliminate the possibility of clashes for practical purposes.

Encoding wxh in the alias name is never a good idea, as the definition of the alias will usually have a different wxh from ICEkit's default, and we'd either need to live with a misleading alias name, or change all the templates to use a non-misleading name.